distribution/conf/env-development/styx-config.yml

```diff
@@ -1,5 +1,3 @@
-include: ${STYX_HOME:classpath:}/conf/default.yml
-
```

Comment:
Not sure why this is removed?

I understand we will still support configuration inheritance, and this would have served at least as an example how it is used?